Makefile

```diff
@@ -310,10 +311,12 @@ catalog-build: opm ## Build a catalog image.
 	echo "FROM quay.io/operator-framework/opm:$(OPM_VERSION)-ppc64le" >> $(TMP_DIR).Dockerfile
 	cat catalog/Dockerfile_final_stage >> $(TMP_DIR).Dockerfile
     ## Building catalog image
-	$(CONTAINER_TOOL) build -f $(TMP_DIR).Dockerfile -t $(CATALOG_IMG) .
+	$(CONTAINER_TOOL) build -f $(TMP_DIR).Dockerfile -t $(CATALOG_IMG) -t $(CATALOG_IMG_LATEST) .
 	rm -rf $(TMP_DIR)
+	rm -rf $(TMP_DIR).Dockerfile
```

Comment:
If you are removing the directory itself in the above command, why do you need to delete this files explicitly?

Reply:
It creates both a temporary directory and a temporary Dockerfile in the current directory.

Reply:
oops, my bad.. 🤦🏽‍♂️